Tools/Scripts/webkitpy/w3c/test_importer.py

```diff
@@ -621,6 +620,12 @@ def import_tests(self):
         for prefixed_value in sorted(total_prefixed_property_values, key=lambda p: total_prefixed_property_values[p]):
             _log.info('  %s: %s', prefixed_value, total_prefixed_property_values[prefixed_value])
 
+        # Fetch the Upstream commit ID for the WPT import So it is ready to insert in the commit message.
+        upstream_id = self.test_downloader().upstream_commitID_head
```

Comment:
We call this an "upstream commit" in the output, so why is this an "upstream_id" here?

This means this PR now calls this value three things: "upstream commit", "upstream_id", and "upstrema_commitID_head"—can we just use a single name for this?